packages/sanity/src/core/studio/components/navbar/userMenu/UserMenu.tsx

```diff
@@ -114,44 +104,7 @@ export function UserMenu() {
             </Flex>
           </Card>
 
-          {!theme?.__legacy && (
-            <>
-              <MenuDivider />
-
-              <Box padding={2}>
-                <Label size={1} muted>
-                  Appearance
-                </Label>
-              </Box>
-
-              <MenuItem
-                aria-label="Use system appearance"
-                icon={DesktopIcon}
-                onClick={clearStoredScheme}
-                pressed={usingSystemScheme}
-                text="System"
-                iconRight={usingSystemScheme && <CheckmarkIcon />}
-              />
-
-              <MenuItem
-                aria-label="Use dark appearance"
-                icon={MoonIcon}
-                onClick={() => setScheme('dark')}
```

Comment:
minor: we should try and avoid using inline functions (eslint should display a warning here)